core/services/feeds/service.go

```diff
@@ -894,12 +894,8 @@ func (s *service) ApproveSpec(ctx context.Context, id int64, force bool) error {
 					return fmt.Errorf("failed while checking for existing ccip job: %w", txerr)
 				}
 			case job.StandardCapabilities:
-				existingJobID, txerr = tx.jobORM.FindStandardCapabilityJobID(ctx, *j.StandardCapabilitiesSpec)
```

Comment:
Does this also need a fix internally?
It's doing
`INNER JOIN standardcapabilities_specs sc on jobs.standard_capabilities_spec_id = sc.id AND sc.command = $1`

Shouldn't StandardCapabilitiesSpec also have the capability ID? Then it would make sense to not allow multiple jobs with the same cap ID.

Reply:
Yep that is likely the right long-term fix 👍

Reply:
I would like to merge this to get testnet unblocked and then we can re-evaluate whether we do the proper  fix now or later